src/components/Message/MessageSimple/MessageTextContainer.js

```diff
@@ -9,29 +9,29 @@ const TextContainer = styled.View`
   border-bottom-left-radius: ${({ theme, groupStyle }) =>
     groupStyle.indexOf('left') !== -1
       ? theme.message.content.textContainer.borderRadiusS
-      : theme.message.content.textContainer.borderRadiusL};
+      : theme.message.content.textContainer.borderRadiusL}px;
   border-bottom-right-radius: ${({ theme, groupStyle }) =>
     groupStyle.indexOf('right') !== -1
       ? theme.message.content.textContainer.borderRadiusS
-      : theme.message.content.textContainer.borderRadiusL};
+      : theme.message.content.textContainer.borderRadiusL}px;
   border-top-left-radius: ${({ theme, groupStyle }) =>
     groupStyle === 'leftBottom' || groupStyle === 'leftMiddle'
       ? theme.message.content.textContainer.borderRadiusS
-      : theme.message.content.textContainer.borderRadiusL};
+      : theme.message.content.textContainer.borderRadiusL}px;
   border-top-right-radius: ${({ theme, groupStyle }) =>
     groupStyle === 'rightBottom' || groupStyle === 'rightMiddle'
       ? theme.message.content.textContainer.borderRadiusS
-      : theme.message.content.textContainer.borderRadiusL};
+      : theme.message.content.textContainer.borderRadiusL}px;
   padding: 5px;
-  padding-left: 8;
-  margin-top: 2;
-  padding-right: 8;
+  padding-left: 8px;
+  margin-top: 2px;
+  padding-right: 8px;
```

Comment:
also could change the padding on the line above to be `padding-vertical: 5px;`. also, we're not dealing with styled-component sort keys?
```suggestion
  margin-top: 2px;
  padding-horizontal: 8px;
```

Reply:
added a sorter to the eslint for TS so we can fix them as we convert.